### ISSUE TITLE
UEFI pc

### ISSUE THREAD

**Issue by UndeadHeavy** [BlackArch/blackarch#1963]
Does Blackarch support UEFI/efi pc? With gpt partition? If so, what is loader location for grub/bootloader ? "\EFI\???\???.efi"

**Comment by ikstream** [comment]
Yes, Blackarch does work on UEFI systems. The installer on the other hand is not yet configured to do so. You would have to install it manually.

**Comment by UndeadHeavy** [comment]
The installer on the other hand? What does it means?

**Comment by ikstream** [comment]
The installer of blackarch does not support UEFI so far.

**Comment by UndeadHeavy** [comment]
What ? So how can we install it on uefi? Im totally confused

**Comment by ikstream** [comment]
Everything the installer does(simplified) is installing arch linux and add some blackarch related stuff.
So you could install arch linux manually and add BA on top of it. There is a guide for it out there as well.
You could also use manjaro,antergos or arch anywhere if their installer supports UEFI installation and add BA to it

**Comment by UndeadHeavy** [comment]
Its not helping
